### PR TITLE
feat: integrate History plugin

### PR DESCRIPTION
**Pull request by TomasSlama**
[FX-4123]

### Description

Integrate undo/redo functionality

### How to test

- play with temploy, write some text and try to use CTRL + Z

### Development checks

- Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- Make sure that additions and changes on design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- Annotate all `props` in component with documentation
- Create `examples` for component
- Ensure that deployed demo has expected results and good examples
- Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- Covered with tests

**Breaking change**

- codemod is created and showcased in the changeset
- test alpha package of Picasso in StaffPortal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-4123]: https://toptal-core.atlassian.net/browse/FX-4123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by TomasSlama** [comment]
@toptal-anvil ping reviewers

**Review by dmaklygin — COMMENTED** [review]
Are you going to test this with cypress or unit tests?

**Comment by TomasSlama** [comment]
@dmaklygin hm do you think we should test Lexical plugin?

**Review by dmaklygin — APPROVED** [review]
Checked on temploy. Works!

**Review by dmaklygin — APPROVED** [review]

File: cypress/component/RichTextEditor.spec.tsx [hunk 1]
```diff
@@ -87,6 +87,31 @@ const setAliases = () => {
...
+    it('correctly uses editor history', () => {
+      cy.mount(renderEditor(defaultProps))
+      setAliases()
+
+      cy.get('@editor').type('f')
+
+      if (isOn('mac')) {
+        cy.get('@editor').type('{cmd}z')
+      } else {
+        cy.get('@editor').type('{ctrl}z')
```
Comment (OleksandrNechai) [inline cypress/component/RichTextEditor.spec.tsx:100]: its command for my mac

**Review by OleksandrNechai — APPROVED** [review]
Tested locally. Works for me although I don't understand how, sometimes it ctrl+Z one word, sometimes two or whole phrase. But I guess this is standard behaviour, I see the same in slack too.

**Pull request merged**